### DISCUSSION TITLE
Segments disappears after every command

### DISCUSSION THREAD

**Discussion by PiotrKlimisz** [JanDeDobbeleer/oh-my-posh#4292] [Troubleshoot] [answered]
Hi there. 

I am sorry if it was somewhere in the issues or discussions, I've really tried founding it but no success. 
I have installed oh-my-posh to windows terminal (powershell) on my new machine exactly as docs says, applied the theme I wanted and noticed that after every command, ctrl+c, git status or just enter the segments are disappearing. I think it should remain as on second screenshot. 

![image](https://github.com/JanDeDobbeleer/oh-my-posh/assets/18614869/5d59a928-ede0-4ec4-a123-e997ee042fa5)
![image](https://github.com/JanDeDobbeleer/oh-my-posh/assets/18614869/0050a6a5-f213-4cac-859e-b7ee0ecf7764)

Do you have any idea if there is something I missed installing it or if there is a bug somewhere?

Thanks in advance!

**Comment by JanDeDobbeleer** [comment, marked as answer]
@PiotrKlimisz this is the [transient prompt feature](https://ohmyposh.dev/docs/configuration/transient).

**Reply by PiotrKlimisz** [reply]
Thanks a lot!
